app/org/sagebionetworks/bridge/services/email/EmailSignInEmailProvider.java

```diff
@@ -16,31 +20,38 @@
 
     private final Study study;
     private final String recipientEmail;
-    private final String subject;
-    private final String body;
     private final String token;
     
-    public EmailSignInEmailProvider(Study study, String recipientEmail, String subject, String body, String token) {
+    public EmailSignInEmailProvider(Study study, String recipientEmail, String token) {
         this.study = study;
         this.recipientEmail = recipientEmail;
-        this.subject = subject;
-        this.body = body;
         this.token = token;
     }
     
     @Override
     public MimeTypeEmail getMimeTypeEmail() throws MessagingException {
+        String encodedRecipientEmail = null;
+        try {
+            encodedRecipientEmail = URLEncoder.encode(recipientEmail, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            throw new BadRequestException("Email was malformed: " + recipientEmail + ".");
+        }
```

Comment:
UnsupportedEncodingException = this never happens if you use UTF-8, error is wrong here as well.